src/transformers/generation/utils.py

```diff
@@ -4300,7 +4300,6 @@ def constrained_beam_search(
     def assisted_decoding(
         self,
         input_ids: torch.LongTensor,
-        assistant_model: Optional["PreTrainedModel"] = None,
```

Comment:
removed according to the deprecation cycle -> remove all references and docstrings